docs/pbc/all/payment-service-providers/payone/integrate-payone.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Integrate Payone
+description: Learn how you can integrate the Payone app into your Spryker shop
+template: howto-guide-template
+---
+
+To integrate Payone, follow these steps:
+
+1. In your store's Back Office, go to **Apps&nbsp;<span aria-label="and then">></span> Catalog**.
+2. Click **Payone**.
+   This takes you to the Payone app details page.
+3. In the top right corner of the Payone app details page, click **Connect app**.
+   This displays a message about the successful connection of the app to your SCCOS. The Payone app's status changes to *Connection pending*.   
+4. Go to [Payone](https://www.payone.com?ref=spryker-documentation) and obtain the credentials.
+   
+   {% info_block infoBox "Info" %}
+
+   It takes some time to obtain credentials from Payone because you have to go through a thorough vetting process by Payone, such as the "know your customer" (KYC) process before Payone verifies you.
+
+   {% endinfo_block %}
+
+5. Go back to your store's Back Office, to the Payone app details page.
```

Comment:
Since there is a time gap due to the verification, consider splitting the instructions into two parts, the second one starting from this step. You can call them `Connect Payone` and `Configure Payone`.